modules/ROOT/pages/amazon/amazon-dynamodb-connector-reference.adoc

```diff
@@ -1606,10 +1599,10 @@ Amazon DynamoDB Connector v1.3.0
 [[repeatable-file-store-iterable]]
 === Repeatable File Store Iterable
 
-[%header%autowidth.spread]
+[%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| Max In Memory Size a| Number | The maximum amount of instances that will be kept in memory. If more than that is required, then it will start to buffer the content on disk. |  | 
+| Max In Memory Size a| Number | The maximum amount of instances to keep in memory. If more are required, then the Runtime  starts to buffer the content on disk. |  | 
```

Comment:
We aren't supposed to use "the Runtime". It's always just Mule or Mule runtime engine (runtime engine no caps)